src/main/java/org/broadinstitute/hellbender/utils/test/MiniClusterUtils.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.utils.test;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.MiniDFSCluster;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.UUID;
+
+public class MiniClusterUtils {
+
+    public static MiniDFSCluster getMiniCluster() throws IOException {
+        final File baseDir = BaseTest.createTempDir("minicluster_storage");
+        final Configuration conf = new Configuration();
+        conf.set(MiniDFSCluster.HDFS_MINIDFS_BASEDIR, baseDir.getAbsolutePath());
+        return new MiniDFSCluster.Builder(conf).build();
+    }
+
+    public static Path getTempPath(MiniDFSCluster cluster, String prefix, String extension) throws IOException {
+        return new Path(cluster.getFileSystem().getWorkingDirectory(), prefix + "_" + UUID.randomUUID() + extension);
```

Comment:
Is there an equivalent to `createTempFile()` you can use here that will guarantee there won't be a name collision?